helpers/metadata_test.go

```diff
@@ -0,0 +1,95 @@
+package helpers
+
+import (
+	tektonv1beta1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	"testing"
+)
+
+var (
+	testObject      = &tektonv1beta1.PipelineRun{}
+	testAnnotations = map[string]string{
+		"a.test/annotation":              "foo",
+		"another.test/annotation-string": "bar",
+	}
+	testLabels = map[string]string{
+		"a.test/label":              "foo",
+		"another.test/label-string": "bar",
+	}
+)
+
+func init() {
+	testObject.SetAnnotations(testAnnotations)
+	testObject.SetLabels(testLabels)
+}
+
+func TestHasAnnotation(t *testing.T) {
+	var tests = []struct {
+		annotation string
+		want       bool
+	}{
+		{"a.test/annotation", true},
+		{"bar", false},
+	}
+
+	t.Logf("Testing that HasAnnotation properly finds object annotations")
+	for _, test := range tests {
```

Comment:
For table driven tests, this variable is called `tt` usually. Kinda like an unwritten rule.

That's it:
```suggestion
	for _, tt := range tests {
```

Reply:
Ack, didn't know that. Fixed, thank you